recipes/_master_package.rb

```diff
@@ -46,13 +46,32 @@
 when 'rhel'
   include_recipe 'yum::default'
 
+  ruby_block "Security off" do
```

Comment:
from a user experience point of view there should be an attribute to control this -- or is security off a definite requirement for running that cookbook at all?